app/cni/pkg/cni/injector_linux.go

```diff
@@ -109,7 +109,7 @@ func mapToConfig(intermediateConfig *IntermediateConfig) (*config.Config, error)
 		if err != nil {
 			return nil, err
 		}
-		cfg.Redirect.Inbound = config.TrafficFlow{
+		cfg.Redirect.Inbound = kumanet_config.TrafficFlow{
```

Comment:
I think we might need to set https://github.com/kumahq/kuma-net/blob/master/transparent-proxy/config/config.go#L16 Enabled to `true`, outbound and inbound. By default should be true, but default for bool is false

Reply:
Haha, we figured this out at the same time as your comment :-D